src/org/zaproxy/zap/extension/pscanrules/CookieSecureFlagScanner.java

```diff
@@ -72,29 +76,53 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
 			iterator.addIterator(cookies2.iterator());
 		}
 
+		List<String> ignoreList = new ArrayList<String>();
```

Comment:
Some of the scan logic could be extracted to a base class and be used by both scanners. Otherwise, same comments as for the previous scanner.